src/LongTermVariability.cpp

```diff
@@ -20,19 +20,19 @@
  |
  *===========================================================================*/
 void LongTermVariability(double h_r1__km, double h_r2__km, double d__km, double f__mhz,
-    double time_percentage, double f_theta_h, double A_T, double *Y_e__db, double *A_Y)
+    double time_percentage, double f_theta_h, double A_T, double *Y_e__db, double *A_Y) 
 {
     // Step 1
     double d_Lq1__km, d_Lq2__km, theta__rad;
     RayTrace(N_9000, h_r1__km, &d_Lq1__km, &theta__rad);
     RayTrace(N_9000, h_r2__km, &d_Lq2__km, &theta__rad);
 
     // Step 2
-    double d_qs__km = 65.0 * pow((100.0 / f__mhz), 1.0 / 3.0);          // [Eqn 184]
-    double d_Lq__km = d_Lq1__km + d_Lq2__km;                            // [Eqn 185]
-    double d_q__km = d_Lq__km + d_qs__km;								// [Eqn 186]
+    double d_qs__km = 65.0 * pow((100.0 / f__mhz), 1.0 / 3.0);          // [Eqn 187]
```

Comment:
Should this use "60" instead of "65"?
![image](https://user-images.githubusercontent.com/40250324/90544532-78dd0d80-e144-11ea-8ec4-f13032f89228.png)